guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-server.adoc

```diff
@@ -9,19 +9,7 @@ Refer to the {InstallingServerDocURL}configuring-{project-context}-server-with-a
 In return, you will receive the {ProjectServer} certificate and CA bundle.
 
 .Procedure
-* Before deploying a renewed custom certificate on your {ProjectServer}, validate the {customssl} input files.
-Note that for the `katello-certs-check` command to work correctly, Common Name (CN) in the certificate must match the FQDN of {ProjectServer}:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# katello-certs-check -t {project-context} \
--b /root/_{project-context}_cert/ca_cert_bundle.pem_ \
--c /root/_{project-context}_cert/{project-context}_cert.pem_ \
--k /root/_{project-context}_cert/{project-context}_cert_key.pem_
-----
-+
-If the command is successful, it returns the following `{foreman-installer}` command.
-You can use this command to deploy the renewed CA certificates to {ProjectServer}:
+. Deploy the renewed CA certificates to {ProjectServer}:
```

Comment:
```suggestion
* Deploy the renewed CA certificates to {ProjectServer}:
```

suggestion: use bullet list. If the procedure only consists of one step, we generally use unordered lists.